packages/usa.michigan/README.md

```diff
@@ -0,0 +1,112 @@
+# 🇺🇸 USA, Michigan
+
+[![License: CC BY SA 3.0](https://img.shields.io/badge/License-CC%20BY%20SA%203.0-blue.svg)](https://creativecommons.org/licenses/by-sa/3.0/)
+
+Contains all the counties of the Michigan state:
+* Alcona
+* Alger
+* Allegan
+* Alpena
+* Antrim
+* Arenac
+* Baraga
+* Barry
+* Bay
+* Benzie
+* Berrien
+* Branch
+* Calhoun
+* Cass
+* Charlevoix
+* Cheboygan
+* Chippewa
+* Clare
+* Clinton
+* Crawford
+* Delta
+* Detroit City
```

Comment:
Why is Detroit City in the list of counties? 🤔

Reply:
Generally in Michigan, Detroit City is considered a separate region from Wayne County because is has a very large population so it's usually separated out as its own region!